ocsci/__init__.py

```diff
@@ -6,3 +6,34 @@
 https://docs.pytest.org/en/latest/reference.html
 under section PYTEST_DONT_REWRITE
 """
+# Use the new python 3.7 dataclass decorator, which provides an object similar
+# to a namedtuple, but allows type enforcement and defining methods.
+from dataclasses import dataclass, field, fields
+
+
+@dataclass
+class Config:
+    DEPLOYMENT: dict = field(default_factory=dict)
+    ENV_DATA: dict = field(default_factory=dict)
```

Comment:
This is a total nitpick, but is there a strong reason to keep this all upper case still?

Reply:
I'd be in favor of moving to lower case, but I'm not sure about the rest of the group

Reply:
I think they should be lower case. We should probably 'reserve' upper case variables for things that come from `constants`, `defaults`, and _maybe_  global vars.

Reply:
I agree. Should we do this in a different PR perhaps?

Reply:
@andrewschoen @zmc yeah let's make that change in another PR, I'd rather not hold this work back for it since it's really just matching what those variables looked like before.